### PR TITLE
fix: [WIP] sketch of an offerTo design taking shape.

### PR DESCRIPTION
**Pull request by erights**
It is different than the `offerTo` we agreed on, though it has many similarities. Altogether I think I like it much better, but we should discuss it --- once it has a bit more shape. I think I'm close to that point.

### CODE REVIEW

File: packages/zoe/src/zoeService/zoeSeat.js [hunk 1]
```diff
@@ -31,35 +58,79 @@ export const makeZoeSeatAdminKit = (
...
+      // `resolve` will likely have no effect since that promise is
+      // already resolved, and the caller has likely already picked it
+      // up. However, if this seat is made internally (by `makeEmptyOffer`
+      // or omitting the seat argument from `mintGains`), then the
+      // offerResult according to the new userSeat will be the first one
+      // for this seat.
+      //
+      // In addition, if the user asks after the seat is forwarded, they'll
+      // get the new offerResult because of the forwarding behavior.
+      offerResultPromiseKit.resolve(E(targetUserSeat).getOfferResult());
```
Comment (Chris-Hibbert) [inline packages/zoe/src/zoeService/zoeSeat.js:95]: Why do we want to resolve this seat's offerResult to the target's?

**Comment by michaelfig** [comment]
Housekeeping: @erights is this PR still relevant?  If not, can you please close it?

**Comment by katelynsills** [comment]
> Housekeeping: @erights is this PR still relevant? If not, can you please close it?

It's still relevant to look at for Zoe design. Potentially we can look at the PR even with this closed, though.

**Comment by michaelfig** [comment]
Closing for now.  I've referenced it from #1909 to keep track of this work.

**Pull request closed without merging**